README.md

```diff
@@ -81,13 +81,13 @@ The project description and base requirements can be found [here][consultancy-ho
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Create instance and class methods on a Rails model that use ActiveRecord methods and helpers<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Write model and feature tests that fully cover data logic and potential user behavior<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Implement a self-referential relationship in ActiveRecord<br>
-&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Utilize Continuous Integration via Travis CI<br>
+&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Utilize Continuous Integration via CircleCI<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Deploy to Heroku<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Implement a production-quality user interface using Bootstrap or other common CSS styling framework<br>
-&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Implement agile project management by using project boards, participating in daily stand-ups and team retros<br>
+&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Implement agile project management via Kanban boards, daily stand-ups, and team retros<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Utilize quality workflow practices: small commits, descriptive pull requests, and code reviews<br>
 &nbsp; &nbsp; ⭐ &nbsp; &nbsp; Organize and refactor code to be more maintainable<br>
-&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Apply RuboCop’s style guide for code quality<br>
+&nbsp; &nbsp; ⭐ &nbsp; &nbsp; Apply Rubocop’s style guide for code quality<br>
```

Comment:
I think we should keep this as RuboCop, similar to their branding: https://rubocop.org/

Reply:
Roger that, will update

Reply:
Updated